app/views/appointments/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<h3>Your appointment</h3>
```

Comment:
index page is for administrator to access. (yes, it should be restricted.)
the users only can see the show page.